custom/icds_reports/ucr/tests/test_child_health_monthly.py

```diff
@@ -193,6 +196,50 @@ def _submit_thr_rations_form(
 
         submit_form_locally(ElementTree.tostring(form), self.domain, **{})
 
+    def _submit_delivery_form(
+            self, form_date, case_id, nutrition_status=None, case_id_2=None):
+
+        form = ElementTree.Element('data')

```

Comment:
wow this looks fun. well done - i would have just been lazy and used a template.